src/components/Cart/CartForm.vue

```diff
@@ -46,25 +48,26 @@ export default {
     ErrorMessage
   },
   setup() {
+    const { t } = useI18n();
```

Comment:
isnt this globally added to vue and can be called with simple $t without importing in each files?